src/lvl_script_value.h

```diff
@@ -25,7 +25,8 @@ extern "C" {
 #endif
 
 void script_process_value(unsigned long var_index, unsigned long plr_range_id, long val2, long val3, long val4, struct ScriptValue *value);
-
+struct Thing* get_creature_in_range_around_any_of_enemy_heart(PlayerNumber plyr_idx, ThingModel crmodel, MapSubtlDelta range);
```

Comment:
guess this one belongs more in thing_list or something as it's not directly scripting logic